src/components/CloseButton.tsx

```diff
@@ -1,19 +1,14 @@
 import React from "react";
 import { CloseIcon } from "@chakra-ui/icons";
-import { Link } from "@chakra-ui/react";
+import { Link, LinkProps } from "@chakra-ui/react";
 import { useRouter } from "next/router";
 
-type CloseButtonProps = {
-    stypeProps?: Record<string, unknown>;
-    onClick?: React.MouseEventHandler;
-};
-
-export const CloseButton: React.FC<CloseButtonProps> = (props) => {
+export const CloseButton: React.FC<LinkProps> = (props) => {
     const router = useRouter();
     const onClick = props.onClick ? props.onClick : () => router.back();
```

Comment:
Thanks for fixing this for me! We can discuss whether it makes more sense to include a default behaviour or if we should just leave onClick as null. I realized that this line might be problematic if we don't want the close button to do anything on click (originally I thought it would be fine since you can override this to whatever you want your `onClick` to be, but I realized you cannot override it to null or else it will automatically be set to `router.back()`

Reply:
yeah i agree, i think it's okay to keep the going back as default, it's mostly used as a back button substitute and you can easily pass href as a prop to change the link